System.IO.Abstractions.TestingHelpers/MockFile.cs

```diff
@@ -571,15 +573,13 @@ public override void Replace(string sourceFileName, string destinationFileName,
                 throw new FileNotFoundException(string.Format(CultureInfo.InvariantCulture, StringResources.Manager.GetString("COULD_NOT_FIND_FILE_EXCEPTION"), destinationFileName));
             }
 
-            var mockFile = new MockFile(mockFileDataAccessor);
-
             if (destinationBackupFileName != null)
             {
-                mockFile.Copy(destinationFileName, destinationBackupFileName, true);
+                Copy(destinationFileName, destinationBackupFileName, true);
```

Comment:
While you're at it please specify the parameter name before the `true`

Reply:
Done